components/mixins/experience-event/industry-verticals/experienceevent-prescription-details.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/industry-verticals/experienceevent-prescription-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Prescription Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["health_and_life_sciences"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "",
+  "definitions": {
+    "prescription-details": {
+      "properties": {
+        "xdm:prescriptions": {
```

Comment:
if you are naming the outer object as "prescriptions" then can we avoid prefixing the properties name with "prescription" ? It will kepe the fully qualified names of the properties short.